src/Engine/UnitActionLifecycles/JoinLifecycle.java

```diff
@@ -29,7 +29,8 @@ public GameActionSet getPossibleActions(GameMap map, Path movePath, Unit actor,
       if( !ignoreResident && resident != null )
       {
         // TODO: Consider if and how off-CO joins should be allowed if tags ever happens
-        if( resident.model.equals(actor.model) && resident != actor && resident.isHurt() )
+        int cargoSize = resident.heldUnits.size() + actor.heldUnits.size();
+        if( resident.model.equals(actor.model) && resident != actor && resident.isHurt() && cargoSize <= resident.model.holdingCapacity )
```

Comment:
Checked the cart on this. They actually just don't let carrying transports participate in joining at all.
That said, I'm fine with this as is because eh... why not?

Reply:
Yeah, I know. But I wanted to.